modules/microshift-uninstall-microshift-rpms.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// microshift_install_rpm/microshift-uninstall.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstall-microshift-rpms_{context}"]
+= Uninstall {microshift-short} from an RPM package
```

Comment:
Procedure headings should start with gerunds ("-ing" words). 
```suggestion
= Uninstalling {microshift-short} from an RPM package
```
Reference: https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html-single/hosted_control_planes/index#hcp-mce-acm-relationship-intro_hcp-overview